builtin/logical/postgresql/backend.go

```diff
@@ -35,6 +35,8 @@ func Backend() *framework.Backend {
 		Secrets: []*framework.Secret{
 			secretCreds(&b),
 		},
+
+		Clean: b.ResetDB,
```

Comment:
ResetDB was there for clean up on configuration change (write to /config) and it made sense to use it as it does exactly what's needed to clean up existing conenctions.